projects/storefrontlib/src/cms-components/order-confirmation/components/order-confirmation-thank-you-message/order-confirmation-thank-you-message.component.html

```diff
@@ -1,5 +1,5 @@
 <ng-container *ngIf="order$ | async as order">
-  <header class="cx-page-header">
+  <header class="cx-page-header" id="cx-thank-you-header">
```

Comment:
I believe that changes in the html DOM is considered a breaking change.  Please use a feature level flag with the version number in which this fix will be released in.
https://sap.github.io/cloud-commerce-spartacus-storefront-docs/feature-flags-and-code-deprecation